src/test/java/io/jenkins/plugins/coverage/model/CoveragePluginITest.java

```diff
@@ -0,0 +1,335 @@
+package io.jenkins.plugins.coverage.model;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Test;
+
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import hudson.model.FreeStyleProject;
+import hudson.model.Result;
+import hudson.model.Run;
+import jenkins.model.ParameterizedJobMixIn.ParameterizedJob;
+
+import io.jenkins.plugins.coverage.CoveragePublisher;
+import io.jenkins.plugins.coverage.adapter.CoberturaReportAdapter;
+import io.jenkins.plugins.coverage.adapter.CoverageAdapter;
+import io.jenkins.plugins.coverage.adapter.JacocoReportAdapter;
+import io.jenkins.plugins.util.IntegrationTestWithJenkinsPerSuite;
+
+import static org.assertj.core.api.Assertions.*;
+
+/**
+ * Integration test for different jacoco and cobertura files.
+ */
+public class CoveragePluginITest extends IntegrationTestWithJenkinsPerSuite {
+
+    private static final String JACOCO_FILE_NAME = "jacoco-analysis-model.xml";
+    private static final String JACOCO_FILE_NAME_2 = "jacoco-codingstyle.xml";
+
+    private static final String COBERTURA_FILE_NAME = "cobertura-higher-coverage.xml";
+    private static final String COBERTURA_FILE_NAME_2 = "coverage-with-lots-of-data.xml";
```

Comment:
This file is one level higher: either copy it to the model package or use path `../coverage-with-lots-of-data.xml`